src/azul/indexer/aggregate.py

```diff
@@ -156,10 +158,34 @@ def get(self) -> list[Any]:
 class SetOfDictAccumulator(SetAccumulator):
     """
     A set accumulator that supports mutable mappings as values.
+
+    >>> from azul.collections import none_safe_itemgetter
+    >>> acc = SetOfDictAccumulator(key=lambda d: d['foo'])
+    >>> d = {'foo': 2}
+    >>> acc.accumulate(d)
+    True
+    >>> acc.accumulate(d)
+    False
+    >>> d = {'foo': 1, 'bar': 1}
+    >>> acc.accumulate(d)
+    True
+    >>> acc.accumulate([d, d])
+    False
+    >>> acc.get()
+    [{'foo': 1, 'bar': 1}, {'foo': 2}]
```

Comment:
```suggestion
    >>> from azul.collections import none_safe_itemgetter
    >>> acc = SetOfDictAccumulator(key=lambda d: d['foo'])
    >>> d = {'foo': 2}
    >>> acc.accumulate(d)
    True
    
    >>> acc.accumulate(d)
    False
    
    >>> d = {'foo': 1, 'bar': 1}
    >>> acc.accumulate(d)
    True
    
    >>> acc.accumulate([d, d])
    False
    
    >>> acc.get()
    [{'foo': 1, 'bar': 1}, {'foo': 2}]
```

Reply:
Agreed at PL that the failing doctest behaves as expected.